client/src/main/com/sinch/sdk/auth/adapters/VerificationApplicationAuthManager.java

```diff
@@ -71,7 +83,7 @@ private String getSignature(
         "\n",
         method,
         bodyMD5Hash,
-        httpContentType,
+        null != httpContentType ? httpContentType : "",
```

Comment:
Do we have cases where there is no content type defined for a request?

Reply:
Yes: for GET requests (e.g. `VerificationStatusByReference`)
So when API is calling the HTTP client (apache for this implementation)
- array list of content type is empty (https://github.com/sinch/sinch-sdk-java/blob/8cdfba59098ef119c85fda207e5a47c5208a462c/openapi-contracts/src/main/com/sinch/sdk/domains/verification/adapters/api/v1/QueryVerificationsApi.java#L105)
- then Content-type header is not present (https://github.com/sinch/sinch-sdk-java/blob/8cdfba59098ef119c85fda207e5a47c5208a462c/client/src/main/com/sinch/sdk/http/HttpClientApache.java#L224)
- then the Auth Manager handle this case

Reply:
Ok, if the serveur is fine with having an empty value for this field, it's fine for me too ;)

Reply:
It make sens not having value: 

- a `GET` request do not have body
- `Content-type` is about how to read body request content

So not required in this context